src/components/NewBeer.js

```diff
@@ -0,0 +1,70 @@
+import './NewBeer.css';
+import { useState } from 'react';
+import axios from 'axios';
+import { Form } from 'react-router-dom';
+
+const NewBeer = ({random}) => {
+
+    const baseUrl = "https://ih-beers-api2.herokuapp.com/beers";
+    const [inputs, setInputs] = useState({});
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        axios
+            .post(`${baseUrl}/new`, inputs)
+            .then(response => {
+                console.log(response)
```

Comment:
After the beer was created sucessfully you could redirect the user to another page (maybe the beers list) or you can have a message saying that the beer was created correctly